debian/sogo.postinst

```diff
@@ -14,6 +14,36 @@ set -e
 # the debian-policy package
 #
 
+# Migration code adapted from original interactive script: sql-update-2.1.17_to_2.3.0-mysql.sh
+
+sqlscript=""
+
+function adjustSchema() {
+    oldIFS="$IFS"
+    IFS=" "
+    part1="`echo -e \"ALTER TABLE $table MODIFY c_partstates mediumtext;\\n\"`";
+    part2="`echo -e \"ALTER TABLE $table ADD COLUMN c_description mediumtext;\\n\"`";
+    sqlscript="$sqlscript$part1$part2"
+    IFS="$oldIFS"
+}
+
+function runDBMigration() {
+    username=$(sogo-tool dump-defaults -f /etc/sogo/sogo.conf | awk -F\" '/ OCSFolderInfoURL =/  {print $2}'|cut -d: -f2|cut -d/ -f3)
+    password=$(sogo-tool dump-defaults -f /etc/sogo/sogo.conf | awk -F\" '/ OCSFolderInfoURL =/  {print $2}'|cut -d: -f3|cut -d@ -f1)
+    hostname=$(sogo-tool dump-defaults -f /etc/sogo/sogo.conf | awk -F\" '/ OCSFolderInfoURL =/  {print $2}'|cut -d@ -f2|cut -d: -f1)
+    database=$(sogo-tool dump-defaults -f /etc/sogo/sogo.conf | awk -F\" '/ OCSFolderInfoURL =/  {print $2}'|cut -d/ -f4)
+    indextable=$(sogo-tool dump-defaults -f /etc/sogo/sogo.conf | awk -F\" '/ OCSFolderInfoURL =/  {print $2}' |  awk -F/ '{print $NF}')
```

Comment:
It's a dependency of the ubuntu-minimal metapackage, not sure if that's enough...